astropy/tests/runner.py

```diff
@@ -334,7 +334,7 @@ def coverage(self, coverage, kwargs):
             warnings.warn(
                 "The coverage option is ignored on run_tests, since it "
                 "can not be made to work in that context.  Use "
-                "'python setup.py test --coverage' instead.",
+                "'pytest --coverage' instead.",
```

Comment:
`--coverage` is an option specific to `setup.py test`, and since this is a warning specific to the test runner it should not be changed ?

Reply:
But we don't allow `setup.py test` anymore. I am confused.

Reply:
yeah I think this should be `pytest --cov` here not `--coverage`

Reply:
@saimn made me undo the changes, so `--cov` doesn't apply no more.

Reply:
This warning is emitted when you run `setup.py test`, so it seems weird that it then refers to pytest instead.
And I'm not sure if `setup.py test` is actually deprecated ?

Reply:
It is not deprecated, it outright won't work anymore as far as this package is concerned.

https://github.com/astropy/astropy/blob/796250381e5f32029bde917050a6f7709e1c86c4/setup.py#L13

Reply:
The test command is still used by downstream packages that use astropy helpers